lib/vanagon/platform/windows.rb

```diff
@@ -81,8 +94,9 @@ def generate_msi_package(project)
         dir_ref = "INSTALLDIR"
         # Actual array of commands to be written to the Makefile
         ["mkdir -p output/#{target_dir}",
-        "mkdir -p $(tempdir)/#{project.name}/staging",
-        "gunzip -c #{project.name}-#{project.version}.tar.gz | '#{@tar}' -C '$(tempdir)/#{project.name}/staging' --strip-components 1 -xf -",
+        "mkdir -p $(tempdir)/{staging,wix}",
+        "cp -v wix/* $(tempdir)/wix",
```

Comment:
The -v was deliberate - just wanted to log what we were copying.
But we will need an -r anyways - we can add that when we need to copy sub-directories (e.g. UI artefacts).